sched/misc/assert.c

```diff
@@ -59,6 +60,12 @@
 #  undef CONFIG_ARCH_USBDUMP
 #endif
 
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+static uint8_t s_last_regs[XCPTCONTEXT_SIZE];
```

Comment:
I think even static global variable still use g_ prefix?
```suggestion
static uint8_t g_last_regs[XCPTCONTEXT_SIZE];
```